app/src/main/res/layout/share_to_apps.xml

```diff
@@ -45,6 +45,7 @@
                 android:layout_height="wrap_content"
                 android:paddingStart="@dimen/share_recent_apps_padding"
                 android:paddingEnd="@dimen/share_recent_apps_padding"
+                android:textColor="?secondaryText"
```

Comment:
I think we want to change the textColor inside of `ShareHeaderTextStyle` instead